src/Microsoft.DotNet.Build.Tasks/PackageFiles/tests.targets

```diff
@@ -281,6 +281,19 @@
             Files="$(TestsSuccessfulSemaphore)" />
   </Target>
 
+  <Target Name="GetDefaultTestRid">
+    <GetTargetMachineInfo>
+      <Output TaskParameter="RuntimeIdentifier" PropertyName="DefaultTestNugetRuntimeId" />
+    </GetTargetMachineInfo>
+    <!-- On Windows, we always use win7-x64 as the default test RID because the build context,
+         usually 32-bit full-framework MSBuild, is not a good default test context. -->
+    <PropertyGroup>
+      <DefaultTestNugetRuntimeId Condition="$(DefaultTestNugetRuntimeId.StartsWith('win'))">win7-x64</DefaultTestNugetRuntimeId>
```

Comment:
* win10-x64 isn't really valid to use on downlevel OS's (8.1, 8, 7)
* win7-x64 is what we are using before the change
* There's no material difference between the two even aside from the above two points. We don't have different assets between win7-x64 and win10-x64 for the scenarios that this default will actually affect.